### PR TITLE
webui: Remove ifIndex from ports list and add debug button for data access

### PR DESCRIPTION
**Pull request by laf**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you signed the [Contributors agreement](http://docs.librenms.org/General/Contributing/) - please do NOT submit a pull request unless you have (signing the agreement in the same pull request is fine). Your commit message for signing the agreement must appear as per the docs.
- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

Removed the ifIndex as discussed here: https://community.librenms.org/t/benefit-of-listing-ifindex-number/467/3

The only way to get the ifIndex after that would be a db lookup so I've added a small tag icon which you can click to get more info.

image

### CODE REVIEW

**Comment by adaniels21487** [comment]
Philosophical question:
Do we want to be displaying debugging output when debug is not enabled?

**Comment by laf** [comment]
It doesn't bother me either way. Easy enough to change it to do that. 

By default it's not shown, you have to click the icon to get the info.

File: html/includes/functions.inc.php [hunk 1]
```diff
@@ -1396,3 +1396,17 @@ function search_oxidized_config($search_in_conf_textbox)
...
+
+/**
+ * @param $data
+ * @return bool|mixed
+ */
+function array_to_htmljson($data)
+{
+    if (is_array($data)) {
+        $data = htmlentities(json_encode($data));
+        return str_replace(',', ',<br />', $data);
```
Comment (murrant) [inline html/includes/functions.inc.php]: I think it would look cleaner without the `,` before the <br />

**Comment by laf** [comment]
trigger ci

**Pull request merged**